airgun/entities/repository.py

```diff
@@ -14,8 +15,22 @@
 
 class RepositoryEntity(BaseEntity):
 
+    def _find_default_http_proxy(self, values):
+        """ Finds the global proxy name and returns updated values accordingly."""
+        proxy = SettingsEntity(self.browser)
+        result = proxy.read(property_name='name = content_default_http_proxy')['table'][0]['Value']
+        global_proxy_name = result.split()[0]
+        if result.split()[0] == 'no':
+            values["repo_content.http_proxy_policy"] = 'Global Default (None)'
+        else:
+            values["repo_content.http_proxy_policy"] = 'Global Default ({})'.format(
+                global_proxy_name)
+        return values
```

Comment:
It really should be `if global_proxy_name == 'no':` in first line.

Also, I personally would find it more readable:
```python
if global_proxy_name == 'no':
    global_proxy_name = 'None'
values["repo_content.http_proxy_policy"] = 'Global Default ({})'.format(
            global_proxy_name)
return values
```

Use whichever you like more, nothing's really wrong with current approach.

Reply:
@mirekdlugosz this looks much better, will use this one.

Reply:
updated PR.